src/timer.js

```diff
@@ -0,0 +1,33 @@
+// import { elHider } from "./visible";
```

Comment:
Тут видимо потерялось расширение файла, поэтому браузер не мог найти файлик по имени
```suggestion
import { elHider } from "./visible.js";
```